app.rb

```diff
@@ -3,9 +3,11 @@
 
 require 'sinatra'
 require 'sinatra/activerecord'
+require 'sinatra/flash'
 require 'action_view'
+require 'sinatra/redirect_with_flash'
 
-%w[policies models routes services presenters].each do |folder_name|
+%w[policies models routes services presenters helpers mailers].each do |folder_name|
   Dir.glob("./#{folder_name}/*.rb") { |f| require f }
```

Comment:
That row is useless if you've written the row below.
Here you load files which are right in the specificated folder, but not nested folders

Reply:
Yeah, I know. The struggle is that there is a trouble with orders in which files are required.
I'll do some researches and find a better approach.